credmark/model/ledger/ledger.py

```diff
@@ -144,3 +149,32 @@ def get_erc20_transfers(self, columns: List[str],
                                      TokenTransferTable,
                                      columns, where, group_by,
                                      order_by, limit, offset)
+
+    def get_block_number_of_date(self, dt_in: Union[datetime, date]):
```

Comment:
change name to: get_block_number_for_date

Use better name for parameters. `timestamp_date` ?

Reply:
renamed